ui/redux/actions/app.js

```diff
@@ -517,6 +517,7 @@ export function doAnalyticsBuffer(uri, bufferData) {
 
 export function doAnalyticsTagSync() {
   return (dispatch, getState) => {
+    console.log('DATS')
```

Comment:
Probably could remove this (it's also the reason the build fails linting)

Reply:
Yes, all of these were troubleshooting duplicate api calls due to another issue in the first file.